_docs/yield-boost-staking-apis.md

```diff
@@ -135,3 +135,47 @@ Example Response:
 - ParachainStaking.DelegationIncreased { delegator: AccountId, candidate: AccountId, amount: Balance }
   - The user's delegation was increased by the specified `amount`
   - [Example block with event](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.turing-staging.oak.tech#/explorer/query/0x485624b3c61626e8f9b7d3dadb168f3b67aa05b8290b2562c836a27643f7427c)
+
+## APR calculation
+
+The goal is to calculate the return of a delegator if one delegates tokens to a particular collator. We start with the fact that
+there is inflation config (inflation_config = [ParachainStaking.InflationConfig](https://github.com/PureStake/moonbeam/blob/master/pallets/parachain-staking/src/inflation.rs#L104)) value that determines how many tokens are issued per round based on amount
+of currently issued tokens (total_issued = Balances.TotalIssued).
+
+Also there is total staked amount (total_staked = [ParachainStaking.Staked](https://github.com/PureStake/moonbeam/blob/master/pallets/parachain-staking/src/lib.rs#L588)) which is updated each round. Having this information one can calculate
+annual return per staked token:
+
+```
+staked_portion = total_staked / total_issued
+annual_return = annual_inflation / staked_portion
+```
+
+For annual_inflation the following logic is applied:
+
+```
+1) annual_inflation = inflation_config.annual.min when total_staked < inflation_config.expect.min;
+2) annual_inflation = inflation_config.annual.max when total_staked > inflation_config.expect.max;
+3) annual_inflation = inflation_config.annual.ideal otherwise;
+```
+
+Then one also need to take into account that part of the tokens goes to parachain
+bond account (par_bond_percent = [ParachainStaking.ParachainBondInfo.percent_of_inflation](https://github.com/PureStake/moonbeam/blob/master/pallets/parachain-staking/src/types.rs#L1567)) and the fact that the lower collator’s stake the higher reward will be given
```

Comment:
This link is moonbeam’s. In our code, are we referencing Moonbeam’s code or have copied over to. OAK-blockchain?